### PR TITLE
Making changes to the documentation and the confirmation redirect

### PR DESCRIPTION
**Pull request by Johnnerz**

### CODE REVIEW

**Comment by Johnnerz** [comment]
Both Hosted and non-Hosted were tested and working locally after the changes have been made

File: simplifycommerce.php [hunk 1]
```diff
@@ -450,7 +450,7 @@ public function processPayment()
 				'?id_cart='.(int)$this->context->cart->id.'&id_module='.(int)$this->id.'&id_order='.
 				(int)$this->currentOrder.'&key='.$this->context->customer->secure_key, '');
 		else
-			Tools::redirect(Link::getPageLink('order-confirmation.php', null, null,
+			Tools::redirect($this->context->link->getPagelink('order-confirmation.php', null, null,
```
Comment (siddii) [inline simplifycommerce.php]: Can you do a text search across the codebase & see if we need to replace it elsewhere?

**Comment by siddii** [comment]
@Johnnerz - Please make sure the save customer flow works fine as well.

File: simplifycommerce.php (hunk 1 above)
Comment (Johnnerz) [inline simplifycommerce.php]: Link::getPageLink is only used in this instance.

Comment (Johnnerz) [inline simplifycommerce.php]: Tested all of the scenarios (hosted/regular/saved card) in HTTPS locally and it works

**Pull request merged**